airbyte-integrations/connectors/source-redshift/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id 'application'
+    id 'airbyte-docker'
+    id 'airbyte-integration-test-java'
+}
+
+application {
+    mainClass = 'io.airbyte.integrations.source.redshift.RedshiftSource'
+}
+
+repositories {
+    maven {
+        url "https://repository.mulesoft.org/nexus/content/repositories/public/"
+    }
+}
+
+dependencies {
+    implementation project(':airbyte-db')
+    implementation project(':airbyte-integrations:bases:base-java')
+    implementation project(':airbyte-protocol:models')
+    implementation project(':airbyte-integrations:connectors:source-jdbc')
+
+    implementation 'com.amazon.redshift:redshift-jdbc42:1.2.43.1067'
```

Comment:
this is what you mentioned in stand up, right? this looks right! if you look at some of the other jdbc sources, they all install whatever driver is specific to the database they support (see mysql and mssql).

Reply:
Yes but in this case, i had to somehow add 
`https://repository.mulesoft.org/nexus/content/repositories/public/` because it is not in the standard maven repo? 
I'm guessing this is fine?

Reply:
yeah. i think so!